dhall-docs/src/Dhall/Docs/Core.hs

```diff
@@ -422,7 +429,7 @@ generateDocs inputDir outLink packageName = do
 -}
 generateDocsPure
     :: Text                    -- ^ Package name
-    -> [(Path Rel File, Text)] -- ^ (Input file, contents)
+    -> [(Path Rel File, ByteString)] -- ^ (Input file, contents)
```

Comment:
Really clever to update this to `ByteString`, that way all filtering is done in a single place.

Reply:
I just obeyed ghc error message :-)